Makefile

```diff
@@ -27,11 +27,17 @@ cover:
   find tests -type d | xargs nosetests --with-coverage --cover-inclusive --cover-erase --cover-package=twilio
 
 docs-install:
-	. venv/bin/activate; pip install twilio sphinx
+	. venv/bin/activate; pip install -U sphinx
 
 docs:
-	. venv/bin/activate; sphinx-apidoc -f twilio -o docs/source
-	cd docs && make clean && make html
+	-rm -rf source/_rst/*
```

Comment:
shouldn't these paths have a `docs/` prefix?

Also, if the directory must exist in order for sphinx to work, I think you'll need to add it yourself. If sphinx is OK with a nonexistent directory, then you can delete the directory itself instead of its contents.

Reply:
Yes, Sphinx is ok with non-existent directory, and it creates one if it doesn't exist. And I will add the docs/ prefix